pkg/commands/objectstore/getkey.go

```diff
@@ -0,0 +1,42 @@
+package objectstore
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v6/fastly"
+)
+
+// GetKeyCommand calls the Fastly API to list the available object stores
```

Comment:
@dgryski is this comment correct? This one and https://github.com/fastly/cli/pull/670/files#diff-5dfcb068798920b452335e73f5eaaadc7d924a049d2dc1a26c14281630770c71R13 seem to be the same as https://github.com/fastly/cli/pull/670/files#diff-1500af34d52eebf6ada35c612be9984f5e4a3ecb3da391d5f36b1ce8779a8f70R13